src/mixxx.cpp

```diff
@@ -1462,7 +1462,13 @@ void MixxxMainWindow::rebootMixxxView() {
     }
 
     setCentralWidget(m_pWidgetParent);
+#ifdef __LINUX__
```

Comment:
What about BSD?

Reply:
so far, the issue has been reported for Ubuntu distros **with X11 only**, thus this ifdef.
And it needs testing on non-X11 to make sure it's no regression there.

And if someone uses mixxx on BSD and stumbles upon this minor UX bug s/he should let us know and we include it here, right ;)

Reply:
Does changing skins still work flawlessly with i3wm for you?